blackberggroup/VA-OAM
# feat: add & link thumb out to YT

## What type of PR is this? (check all applicable)


## Description Add and center thumbnail under copy. Link to YouTube.

## Related Tickets & Documents
- Monday Task https://blackberg-group.monday.com/boards/1931719392/pulses/7714807490

## QA Instructions, Screenshots, Recordings
Ensure thumbnail from news landing page is being used and centered under copy. Ensure it is linked to YouTube video.

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- _site/assets/css/styles.css
 - _site/news-and-events/videos-podcasts/about-oam/index.html
 - assets/css/styles.css
 - news-and-events/videos-podcasts/about-oam.html
